docs/references/general/fitacf.md

```diff
@@ -1,2 +1,169 @@
+| *lagfr*                 | *us* | ***short***  | Lag to first range                                                                      |
+| *smsep*                 | *us* | ***short***  | Sample seperation                                                                       |
+| *ercod*                 | **None**         | ***short***  | Error Code                                                                              |
+| *stat.agc*              | **None**         | ***short***  | AGC status word                                                                         |
+| *stat.lopwr*            | **None**         | ***short***  | LOPWR status word                                                                       |
+| *noise.search*          | **None**         | ***float***  | Calculated noise from clear frequency search                                            |
+| *noise.mean*            | **None**         | ***float***  | Average Noise from clear frequency band                                                 |
+| *channel*               | **None**         | ***short***  | Channel number for variant changes in radars parameters in one scan, ex) frequency      |
+| *bmnum*                 | **None**         | ***short***  | Beam number                                                                             |
+| *bmazm*                 | **None**         | ***float***  | Beam Azimuth                                                                            |
+| *scan*                  | **None**         | ***short***  | Scan flag, switches between 0 to 1 or 0 to -1 to indicate a new scan                    |
+| *offset*                | **None**         | ***short***  | Offset between channels of stereo radars                                                |
+| *rxrise*                | *us* | ***short***  | Receiver rise time                                                                      |
+| *intt.sc*               | *s*        | ***short***  | whole number of seconds of integration time                                             |
+| *intt.us*               | *us* | ***int***    | Factional number of microseconds of integration time                                    |
+| *txpl*                  | *us* | ***short***  | Transmitted pulse length                                                                |
+| *mpinc*                 | *us* | ***short***  | Multi-pulse increment                                                                   |
+| *mppul*                 | **None**         | ***short***  | Number of pulses in the sequence                                                        |
+| *mplgs*                 | **None**         | ***short***  | Number of lags in the sequence                                                          |
+| *mplgexes*              | **None**         | ***short***  |                                                                                         |
```

Comment:
@egthomas @ksterne  do you want to fill this row in?

Reply:
AFAIK, `mplgexs` is not a field in fitacf files, only fitex files.

Reply:
```
dmapdump 20000413.0000.00.gbr.fitacf | less
       short   "mppul" = 7
        short   "mplgs" = 18
        short   "mplgexs" = 0
        short   "ifmode" = 0
        short   "nrang" = 70
```
Then explain this?

Reply:
I don't know why it is in fitacf but it does appear :/

Reply:
It's not defined in [fitblk.h](https://github.com/SuperDARN/rst/blob/develop/codebase/superdarn/src.lib/tk/fitacf.2.5/include/fitblk.h).

But it is defined in [rprm.c](https://github.com/SuperDARN/rst/blame/444abd4f38ff18bb6f9a25b6272254fe8af8b4d0/codebase/superdarn/src.lib/tk/radar.1.22/src/rprm.c#L282), so the "Radar parameter" structure just defines it, regardless of whether or not it makes sense to. The blame history shows it "has always been here", which means it was added by VT when they did the fitex stuff.

Anything with the `ex` suffix is fitex related though, not fitacf. It was needed so that VT could process and fit the `tauscan` pulse sequence modes they were running.

Reply:
Yea, well still in fitacf right now so I would like to still document what it means. Thanks for the information, I will see if it is documented there.

Reply:
Perhaps we can just document `mplgexes` as e.g. "(not used in FITACF file format)"?